docs/builders/googlecompute.mdx

```diff
@@ -261,6 +261,45 @@ https://missionimpossiblecode.io/post/winrm-for-provisioning-close-the-door-on-t
 
 This build can take up to 15 min.
 
+### Windows over WinSSH Example
+
+The following uses Windows SSH as backend communicator
+[https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse](https://docs.microsoft.com/en-us/windows-server/administration/openssh/openssh_install_firstuse)
+
+<Tabs>
+<Tab heading="HCL2">
+
+```hcl
+source "googlecompute" "windows-ssh-example" {
+  project_id = "MY_PROJECT"
+  source_image = "windows-server-2019-dc-v20200813"
+  zone = "us-east4-a"
+  disk_size = 50
+  machine_type = "n1-standard-2"
+  communicator = "ssh"
+  ssh_username = var.packer_username
+  ssh_password = var.packer_user_password
+  ssh_timeout = "1h"
+  metadata = {
+    windows-startup-script-cmd = "net user ${var.packer_username} \"${var.packer_user_password}\" /add /y & wmic UserAccount where Name=\"${var.packer_username}\" set PasswordExpires=False & net localgroup administrators ${var.packer_username} /add & powershell Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0 & powershell Start-Service sshd & powershell Set-Service -Name sshd -StartupType 'Automatic' & powershell New-NetFirewallRule -Name 'OpenSSH-Server-In-TCP' -DisplayName 'OpenSSH Server (sshd)' -Enabled True -Direction Inbound -Protocol TCP -Action Allow -LocalPort 22 & powershell.exe -NoProfile -ExecutionPolicy Bypass -Command \"Set-ExecutionPolicy -ExecutionPolicy bypass -Force\""
+  }
+}
+
+build {
+  sources = ["sources.googlecompute.windows-ssh-example"]
+
+  provisioner "powershell" {
+    script = "../scripts/install-features.ps1"
+    elevated_user     = var.packer_username
+    elevated_password = var.packer_user_password
+  }
+}
+```
+
+</Tab>
+</Tabs>
+
```

Comment:
These should be necessary only if there's another JSON tab, but since there is none, I think we can get rid :)
```suggestion
```